README.md

```diff
@@ -134,6 +135,8 @@ A `json` configuration file of following format is used to configure Grabbit.
 
 #### Optional fields
 
+* __deltaContent__: boolean, where ```false``` has grabbit sync & overwrite all content nodes regardles of date properties, and ```true``` syncs only 'delta' or changed content and leaves unchanged content alone (with deltaContent on, syncs will be significantly faster).
```

Comment:
May be we should also add the fact that the `changed content` is calculated by comparing either of the `jcr:lastModified`, `cq:lastModified` or `jcr:created` Date with the Date when the `path` was successfully synced as recorded by the Grabbit Client.

In other words, we should be explicit that the deltaContent will only make sense if the paths stay the same for subsequent runs.

That is : path - `/a/b/c/d` finished successfully and then we ran `/a/b/c/d` again with `deltaContent : true` then that will work but if we run `/a/b/c/d/e` the second time, `deltaContent : true` will not work since there was no job run for `/a/b/c/d/e` before.